### PR TITLE
Notifikation M117

### LINKED ISSUE #1833: [Q] What should be shown on the Notification screen?
I use M117 notifications from the Octoprint-DetailedProgress plugin to push progress info to my TFT35 screen. This works fine in recent releases. I was also expecting to find a log of these messages in the 'Notifications' screen after clicking this area on the status screen. However this screen is always empty for me.

Is this a configurable option I can enable somewhere, or a bug or maybe by design?

### PR DESCRIPTION
**Pull request by radek8**
Display of M117 messages on the notification screen.
With the option to turn this view on or off in the CONFIG.INI file
Fix M569 error

resolves #1833 

image

### CODE REVIEW

File: Copy to SD Card root directory to update/config.ini [hunk 1]
```diff
@@ -173,6 +173,11 @@ files_list_mode:1
 #     TOAST: A non-blocking Toast notification is displayed for few seconds. No user interaction is needed.
 ack_notification:1
 
+#### Notification M117
+#Display of M117 messages on the notification screen
```
Comment (digant73) [inline Copy to SD Card root directory to update/config.ini:177]: leave a space after # and put a dot at the end. So "# Display..... screen."

**Review by digant73 — CHANGES_REQUESTED** [review]

File: Copy to SD Card root directory to update/config.ini (hunk 1 above)
Comment (radek8) [reply Copy to SD Card root directory to update/config.ini:177]: Thank you

**Comment by Tannoo** [comment]
A question: If `#define NOTIFICATION_M117 0` is disabled, why is it then commented out by default also?

and will `# define` work just as well as `#define`?

`//# define NOTIFICATION_M117 0  // Default 0`

**Comment by radek8** [comment]
> In `config.c`, don't you want a `break;` for the M117 case?

Fixed
Thank you

**Pull request merged**